sql/src/main/java/io/crate/planner/node/dql/CountPhase.java

```diff
@@ -106,4 +108,15 @@ public void writeTo(StreamOutput out) throws IOException {
         whereClause.writeTo(out);
         distributionInfo.writeTo(out);
     }
+
+    public void replaceSymbols(Function<Symbol, Symbol> replaceFunction) {
+        if (whereClause.hasQuery()) {
```

Comment:
Would it make sense to implement a `replaceFunction` or `map(symbol -> symbol)` function on the WhereClause/QueryClause? I think we've this kind of logic in multiple places.